### DISCUSSION TITLE
Submission of Project Proposal

### DISCUSSION THREAD

**Discussion by himanshu1409** [camicroscope/GSOC#35] [General]
Hey @nanli-emory, I have submitted my GSoC Proposal for project idea [4] Collection and Study Management on the GSoC 2023 website. Hoping to get some feedbacks from your side soon. 
Thanks

**Comment by nanli-emory** [comment]
Hi @himanshu1409,  I haven't seen any system design for collection and study management. How do you want to refactor current collection management and what study management do you want to implement?

**Reply by himanshu1409** [reply]
I would like to implement an MVC architecture for the project where the collections will be stored using mongoose. I have explained it in detail using various flowcharts and code snippets. I have also explained how can we improve searching within the collection and validating JSONs for each collection. This is what I was able to comprehend from the project. If you could explain in detail and guide me about what more I can do, I would definitely try to work on that.

**Reply by nanli-emory** [reply]
Please read the project idea carefully. ```As the slides and annotations grows, researchers and pathologists need to an efficient way to organize slides and annotations. Currently, caMicroscope use the concept of collection to manage slides.``` the concept of collection is not MongoDB collection. It is the entity to manage the slides.
